core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -335,54 +368,113 @@ <h2 class="oppia-fractions-landing-h2 oppia-fractions-landing-centered-h2" style
     .oppia-fractions-landing-image {
       position: relative;
       z-index: 20;
-      max-width: 80%;
+      max-width: 100%;
       max-height: 500px;
     }
     .oppia-fractions-landing-image-mobile {
       display: none;
     }
+    .video-frame {
+      position: relative;
+      width: 270px;
+      height: 600px;
+      margin: auto;
+      border: 11px black solid;
+      border-top-width: 50px;
+      border-bottom-width: 50px;
+      border-radius: 36px;
+      background: #000000;
+    }
+    .video-frame:before {
+      content: '';
+      display: block;
+      width: 60px;
+      height: 5px;
+      position: absolute;
+      top: -25px;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      background: #333;
+      border-radius: 10px;
+    }
+    .video-frame:after {
+      content: '';
+      display: block;
+      width: 35px;
+      height: 35px;
+      position: absolute;
+      left: 50%;
+      bottom: -58px;
+      transform: translate(-50%, -50%);
+      background: #333;
+      border-radius: 50%;
+    }
+
     @media screen and (max-width: 768px) {
       .oppia-fractions-landing-section-inner,
       .oppia-fractions-landing-section {
         height: auto;
       }
       .oppia-fractions-landing-h1 {
         font-size: 1.6em;
-        margin-left: 20%;
+        margin-left: 10%;
         margin-right: 20%;
-        padding-bottom: 30px;
         padding-left: 0 !important;
         padding-right: 0 !important;
         padding-top: 40px;
         text-align: center !important;
-        width: 60%;
+        width: 80%;
+      }
+
+      .easy-to-follow {
+        white-space: pre-wrap;
       }
       .oppia-fractions-landing-h2 {
         font-size: 1em;
-        text-align: left !important;
+        text-align: center !important;
```

Comment:
Is it possible at all to get the desired effect without using !important, here and above? In general we try to avoid it.